src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R5/Immunization.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "Immunization",
```

Comment:
minor, but you can put R4 version into normative, and keep only stu3 version. That way you will add only two files instead of three.

Reply:
I left it, just in case we need to customize.